students/16th/jinyounglim/js/main.js

```diff
@@ -0,0 +1,45 @@
+const btnTxtAll = document.querySelector('.btn_txt_all');
+const feedTxt = document.querySelector('.feeds_txt p');
+
+btnTxtAll.addEventListener('click', txtShow = () => {
```

Comment:
이벤트에서 함수를 정의할 때는 익명함수로도 충분합니다!
```suggestion
btnTxtAll.addEventListener('click', () => {
```